scanpy/_utils.py

```diff
@@ -98,11 +99,11 @@ def deprecated_arg_names(arg_mapping: Mapping[str, str]):
     arg_mapping
         Mapping from deprecated argument name to current argument name.
     """
+
     def decorator(func):
         @wraps(func)
         def func_wrapper(*args, **kwargs):
-            warnings.simplefilter(
-                'always', DeprecationWarning)  # turn off filter
+            warnings.simplefilter('always', DeprecationWarning)  # turn off filter
             for old, new in arg_mapping.items():
```

Comment:
Why is this being reformatted?

Reply:
Also, could you point me to the code you changed in this file? It's hard to tell with all the formatting changes.

Reply:
Yes, sorry for that. The formating changes are only in _utils.py, the other changed files have actual code changes.

Reply:
Cool, good to know.

I feel like we should probably start splitting this up into multiple files, but that'll be in another PR.